images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -275,7 +281,7 @@ function Get-PowerShellModules {
     $result += (Get-ToolsetContent).powershellModules.name | Sort-Object | ForEach-Object {
         $moduleName = $_
         $moduleVersions = Get-Module -Name $moduleName -ListAvailable | Select-Object -ExpandProperty Version | Sort-Object -Unique
-        return "$($moduleName): $($moduleVersions -join ', ')"
+        return [ToolVersionsListNode]::new($($moduleName), $($moduleVersions), '^.+', "Inline")
```

Comment:
Let's use `^\d+` regex for all other PowerShell modules. See Linux: https://github.com/actions/runner-images/blob/main/images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1#L283

Reply:
agree, corrected